source/trees/sec-trees-in-action.ptx

```diff
@@ -211,4 +209,10 @@
             </input>
         </sage>
     </subsection>
+    <subsection xml:id="subsec-Conclusion">
+        <title>Conclusion</title>
+        <p>
+            In this exercise, we only optimized construction material costs. In a real-world scenario, we may want to create a railway that optimizes both travel time and material costs. In the case of the Chicago L train system, the railway resembles a tree when ignoring the downtown Loop. The L receives criticism for its lack of interconnectivity. For example, finding an efficient route connecting the end of the Blue Line with the end of the Red Line is challenging because a traveler may need to commute all the way downtown from one end of the railway to reach another end. As an interesting challenge, you can optimize both travel time and construction costs.
```

Comment:
I'm not sure if we had a rule regarding the use of the "we" pronoun. I have had a remark about it to avoid it by rephrasing the sentence. I'm just pointing this out as we may need to revisit and decide what the rule is about it and make changes in other chapters if needed.